src/module/display.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉